providers/shared/components/logstore/id.ftl

```diff
@@ -0,0 +1,59 @@
+[#ftl]
+
+[@addComponent
+    type=LOGSTORE_COMPONENT_TYPE
+    properties=
+        [
+            {
+                "Type" : "Description",
+                "Value" : "Defines a standalong log store used to capture function base logs"
+            }
+        ]
+    attributes=
```

Comment:
Should this have an engine attribute? Feels like something where the implementations will vary considerably.